### DISCUSSION TITLE
How to pass in the options for the "ferium profile create" command

### DISCUSSION THREAD

**Discussion by TechPro424** [gorilla-devs/ferium#117] [Q&A] [answered]
I'm trying to create a script that automatically installs Ferium and adds some mods, but as of now, I do not know any way to automatically pass in the required values (such as output directory and Minecraft version) to the `ferium profile create` command. Is there a way to do that? (Maybe by passing flags such as `--minecraft-version 1.18.2` or something?)

**Comment by JustSimplyKyle** [comment]
Yes!
`--name test`
`--game-version 1.12.2`
`--mod-loader forge`
`--output-dir /outputting`
These are the flags!
~~I found them in the integration tests~~

**Reply by TechPro424** [reply]
Thanks!

**Reply by TechPro424** [reply]
1 more question: How do I make the output dir stay the default? (I don't want to pass in a custom output dir, I want it to use the default)

**Reply by theRookieCoder** [reply]
> 1 more question: How do I make the output dir stay the default? (I don't want to pass in a custom output dir, I want it to use the default)

Oh yeah that should be allowed, I'll make an issue for that (edit: #118)

**Comment by theRookieCoder** [comment, marked as answer]
`ferium profile create help`

```text
Create a new profile
Optionally provide all the options, to create the profile without the UI
Use the import flag to import mods from another profile

USAGE:
    ferium profile create [OPTIONS]

OPTIONS:
        --game-version <GAME_VERSION>
            The Minecraft version to check compatibility for

        --import [<IMPORT>]
            Copy over the mods from an existing profile
            Optionally, provide the name of the profile to import mods from

        --mod-loader <MOD_LOADER>
            The mod loader to check compatibility for [possible values: quilt, fabric, forge]

        --name <NAME>
            The name of the profile

        --output-dir <OUTPUT_DIR>
            The directory to output mods to
```
I've removed some irrelevant information

**Reply by TechPro424** [reply]
Now that #118 is closed, how do I use the default output directory in the flags?
Is it something like `--output-dir default` or do I not need to use the flag at all and it automatically selects the default dir without asking the user to input it?

**Reply by theRookieCoder** [reply]
You can just not provide the flag and it will use the default one, but this is only for using flags

**Reply by TechPro424** [reply]
Thanks!
